.gitignore

```diff
@@ -17,4 +17,5 @@ app/src/main/res/values/developer-config.xml
 app/src/androidTest/java/com/mapbox/maps/testapp/activity
 sdk/src/androidTest/res/values/developer-config.xml
 extension-style-app/src/main/res/values/developer-config.xml
+sdk/src/main/res/values/developer-config.xml
```

Comment:
Why `developer-config.xml` appeared in `sdk` module?

Reply:
access token is needed to compile the instrumented test in sdk module, I added the token locally to fix the build, but found that we didn't have the gitignore config setup for the file, so added here.

Reply:
It's just for not accidentally include the token with the commit, so only relevant to local build. Since ci doesn't push code to our codebase, this doesn't matter.